examples/update_rawdog_config.py

```diff
@@ -0,0 +1,52 @@
+    },
+    {
+        "role": "user",
```

Comment:
16-18 look like a duplicate (of 20-22)? I checked and it doesn't seem to be happening automatically.

Reply:
Weird. I think this script was made before my change to llm_handler was finalized. I'll fix it.